### DISCUSSION TITLE
[@types/mailchimp__mailchimp_marketing]: invalid return type for `mailchimp.ping.get()`

### DISCUSSION THREAD

**Discussion by nbouvrette** [DefinitelyTyped/DefinitelyTyped#65512] [Issues with a @types package]
I was just trying to use `mailchimp.ping.get()` and noticed that the type was set to `mailchimp.ping.APIHealthStatus | mailchimp.ErrorResponse`

In fact, when you call `mailchimp.ping.get()` it returns a `Promise` so I had to override the type like this in the meantime:

```ts
  const result = await (mailchimp.ping.get() as unknown as Promise<
    ReturnType<typeof mailchimp.ping.get>
  >)
```

[a comment by an automated account is not shown]

**Comment by hkleungai** [comment]
It may be a late reply, but please check https://github.com/DefinitelyTyped/DefinitelyTyped/pull/71464 that I just submitted.
